src/Extractor/DibiSupportExtractorTrait.php

```diff
@@ -1,174 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Keboola\DbExtractor\Extractor;
-
-use ErrorException;
-use Dibi\Connection;
-use Dibi\Exception as DibiException;
-use Keboola\Csv\CsvWriter;
-use Keboola\Csv\Exception as CsvException;
-use Keboola\DbExtractor\Exception\ApplicationException;
-use Keboola\DbExtractor\Exception\DeadConnectionException;
-use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
-use Psr\Log\LoggerInterface;
-use Throwable;
-use Dibi\Result;
-use Keboola\DbExtractor\DbRetryProxy;
-use Keboola\DbExtractor\Exception\UserException;
-
-/**
- * This trait solves compatibility problem:
- * - Extractor class from db-extractor-common depends on PDO
- * - PDO ODBC driver (DIBI -> PDO -> ODBC -> DB) doesn't work with special types (MAP, ARRAY, ...) -> it returns NULL
- * - ODBC driver (DIBI -> ODBC -> DB) works with special types
- * - ... therefore must be used ODBC driver and fixed compatibility with db-extractor-common package
- */
-trait DibiSupportExtractorTrait
```

Comment:
Povodne bol db-extractor-common zavisli na PDO, .. tento trait pridaval podporuju pre ODBC/DiBi, tak ze bolo potrebne vela metod pretypovat.

Konecne to mozeme zmazat, lebo uz mame `db-extractor-adapter`.